integration-tests/smoke/ccip/ccip_batching_test.go

```diff
@@ -226,6 +229,77 @@ func Test_CCIPBatching(t *testing.T) {
 				require.Equal(t, changeset.EXECUTION_STATE_SUCCESS, state)
 			}
 		}
+
+		// update the start and end seq nums
+		for _, srcChain := range sourceChains {
+			startSeqNum[srcChain] = startSeqNum[srcChain] + numMessages
+		}
+	})
+
+	t.Run("max evm batch size", func(t *testing.T) {
+		var (
+			sourceChain = sourceChain1
+			otherSender = mustNewTransactor(t, e.Env.Chains[sourceChain])
+			transactors = []*bind.TransactOpts{
+				e.Env.Chains[sourceChain].DeployerKey,
+				otherSender,
+			}
+			errs = make(chan error, len(transactors))
```

Comment:
I guess channel's size should be 0 or 1 https://github.com/uber-go/guide/blob/master/style.md#channel-size-is-one-or-none